package.json

```diff
@@ -10,7 +10,19 @@
     "test": "react-scripts test --watchAll=false",
     "test:coverage": "react-scripts test --coverage --watchAll=false",
     "test:update": "react-scripts test --updateSnapshot --watchAll=false",
-    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls"
+    "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o docs"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/*.{js,jsx}",
+      "!**/node_modules/**",
+      "!**/*.stories.{js,jsx}",
+      "!**/index.{js,jsx}",
+      "!**/config/**",
+      "!**/pages/**"
```

Comment:
I think this setting still has problem
- We need to cover pages folder, but we dont want to cover the index.js file inside the component and pages folder, is this works?
```!pages/**/index.{js,jsx} !components/**/index.{js,jsx} ```

Reply:
My mistake. I though you said pages was not going to be covered, for some reason. If that's the case, the line under stories takes care of all index.js files